playbooks/roles/edxapp/defaults/main.yml

```diff
@@ -381,7 +381,6 @@ EDXAPP_FEATURES: &edxapp_features
   ENABLE_DJANGO_ADMIN_SITE: true
   ENABLE_ENROLLMENT_EMAIL: "{{ EDXAPP_FEATURES_ENABLE_ENROLLMENT_EMAIL }}"
   ENABLE_EXPORT_GIT: "{{ EDXAPP_FEATURES_ENABLE_EXPORT_GIT }}"
-  ENABLE_FEEDBACK_SUBMISSION: true
```

Comment:
Wait, do we really want to remove the variable entirely? Should we just set it to false?
Did we add this feature or was it added upstream?